pulp/overlays/moc/smaug/opf-pulp.yaml

```diff
@@ -63,10 +64,9 @@ spec:
       - /tmp
     allowed_import_paths:
       - /tmp
-  route_annotations:
+  ingress_annotations:
     kubernetes.io/tls-acme: "true"
```

Comment:
do I need to do another thing to enable the cert?

Reply:
I think it would be enough.

Reply:
What do you want to achieve with this annotation? To generate a cert? I don't think that can work... That would assume the `acme-operator` can act upon Ingres.. but you want cert manager to solve this.. since you need a single cert, the same one, shared for all the routes.

Reply:
Correct, i guess that why the idea was to use ingress instead of multiple routes, to have one ingress with all the paths, as the host is the same.
and acme seems to be able to handle ingress : https://github.com/tnozicka/openshift-acme/blob/master/docs/design/architecture.adoc#managed-objects